source/common/quic/BUILD

```diff
@@ -70,7 +70,10 @@ envoy_cc_library(
 envoy_cc_library(
     name = "quic_stat_names_lib",
     srcs = ["quic_stat_names.cc"],
-    hdrs = ["quic_stat_names.h"],
+    hdrs = [
+        "quic_context.h",
```

Comment:
nit: I think it would be a little nicer to separate quic_context_lib but that would probably add a few BUILD files to the PR. I'm fine with checking this in and iterating, or you can go ahead and do it now; up to you. WDYT for now of at least adding a comment with a TODO?

Reply:
+1 for me.